vm/inc/ubpf.h

```diff
@@ -174,7 +174,7 @@ extern "C"
      * @retval -1 Failure.
      */
     int
-    ubpf_load_elf(struct ubpf_vm* vm, const void* elf, size_t elf_len, char** errmsg);
+    ubpf_load_elf(struct ubpf_vm* vm, const void* elf, size_t elf_len, const char* main_section_name, char** errmsg);
```

Comment:
This is a breaking change. Can we maybe create a wrapper that has the old function signature and rename this to something else? I.e. existing programs that call the old api should continue to work?

Reply:
In Microsoft's honor, I added a new function `ubpf_load_elf_ex` with the additional parameter for the entry point. I am sure that there is a better name but I simply could not come up with it. Let me know if the implementation strategy (`ubpf_load_elf` simply pawns off the work to the `_ex` version) is okay!

Reply:
Perfect.